core/conveyor.go

```diff
@@ -74,3 +74,9 @@ type ConveyorFuture interface {
 	// Cancel closes all channels and cleans up underlying structures.
 	Cancel()
 }
+
+// ConveyorPendingMessage is message for conveyor which can pend for response
+type ConveyorPendingMessage struct {
```

Comment:
why do you move this struct from messagebus?

Reply:
Because I use it in ResponseAdapter and don't want to import message buss there